exp/api/v1alpha4/azuremanagedmachinepool_webhook.go

```diff
@@ -86,6 +87,14 @@ func (r *AzureManagedMachinePool) ValidateUpdate(oldRaw runtime.Object, client c
 		}
 	}
 
+	if !reflect.DeepEqual(r.Spec.AvailabilityZones, old.Spec.AvailabilityZones) {
```

Comment:
forgive my naivety, does this handle slice ordering?

Reply:
UT would be nice :)

Reply:
yes, relect.DeepEqual handles slice ordering.

what does UT stand for?

Reply:
unit test

Reply:
@alexeldeib I guess I should make a manual check here right ? WDYT ?

Reply:
can we just add a test that makes sure it works as expected when the slice order changes